src/bbtag/subtags/user/userboostdate.ts

```diff
@@ -6,7 +6,7 @@ import { CompiledSubtag } from '../../compilation';
 import { BBTagRuntimeError, UserNotFoundError } from '../../errors';
 import { SubtagType } from '../../utils';
 
-export class UserBoostDataSubtag extends CompiledSubtag {
+export class UserBoostDateSubtag extends CompiledSubtag {
```

Comment:
Why? I don't even see the change here 😂

Reply:
UserBoost ***Data*** vs UserBoost ***Date*** 🙂